oci/config.py

```diff
@@ -0,0 +1,46 @@
+from collections import namedtuple
+import ConfigParser
+import os
+
+gerrit_default_host = 'gerrit.ovirt.org'
+jenkins_default_host = 'jenkins.ovirt.org'
+
+
+def load():
+    config_dir = os.path.join(os.path.expanduser('~'), '.config/oci.conf')
+    config = ConfigParser.RawConfigParser(allow_no_value=True)
+
+    with open(config_dir, 'r') as config_file:
+        config.readfp(config_file)
```

Comment:
Before we read the actual file, we need to add sections and default values
for all options that have defaults. This ensures that cfg.get() will never fail when
we can use a default value.

Here is a session showing how this can be done:
```python
>>> cfg = configparser.RawConfigParser()
>>> cfg.add_section("gerrit")
>>> cfg.set("gerrit", "host", "defualt.gerrit.org")
>>> cfg.read(os.path.expanduser("~/.config/oci.conf"))
['/home/nsoffer/.config/oci.conf']
>>> cfg.get("gerrit", "host")
'gerrit.ovirt.org'
```

In vdsm we have a big list of sections and options, and a loop adding sections
and setting defaults from this list. We can use similar method here, or just do it
manually since we have only 2 seconds and only one require value in each section.

The code creating a parser with defaults is best extracted to a separate function
so we can do:

    def load():
         cp = config_parser()
         cp.read(...)
         ...

Reply:
Regarding the defaults, I just made them as a global. Since it is also used in the tests, but only has two values, do you think it is worth it to have a file to contain the defaults in?

Reply:
Global constants are fine.